### PR TITLE
Migrate account_compute_tax_amount to 8.0

### PR DESCRIPTION
**Pull request by guewen**
- New API
- A better description of the module
- Do no longer hides the field but set it readonly

### CODE REVIEW

File: account_compute_tax_amount/README.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Recompute tax amount
+====================
+
+The way the tax amount is computed in the core `account` module is prone
+to input errors.
+
+This module forces the the tax amount to always be: `credit - debit`
```
Comment (pedrobaeza) [inline account_compute_tax_amount/README.rst]: double `the`

File: account_compute_tax_amount/README.rst [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Recompute tax amount
+====================
+
+The way the tax amount is computed in the core `account` module is prone
```
Comment (pedrobaeza) [inline account_compute_tax_amount/README.rst]: You should put `in account journal entries`

**Comment by nbessi** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**